UtopiaCity/Controllers/Business/ResumeController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UtopiaCity.Models.Business;
+using UtopiaCity.Services.Business;
+using UtopiaCity.Services.Business.Dtos;
+
+namespace UtopiaCity.Controllers.Business
+{
+    public class ResumeController: Controller
+    {
+        private readonly ResumeAppService _resumeAppService;
+        public ResumeController(ResumeAppService resumeAppService)
+        {
+            _resumeAppService = resumeAppService;
+        }
+
+        public async Task<IActionResult> Index(string filter)
+        {
+            return View("Index", await _resumeAppService.GetAll(filter));
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Create()
+        {
+            ViewData["ResidentAccountId"] = new SelectList(await _resumeAppService.GetAllResidentAccount(), "Id", "FIO");
+            ViewData["ProfessionId"] = new SelectList(await _resumeAppService.GetAllProfession(), "Id", "Name");
+            return View("Create");
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Create(Resume resume)
+        {
+            if (ModelState.IsValid)
+            {
+                await _resumeAppService.Create(resume);
+                return RedirectToAction(nameof(Index));
+            }
+
+            return View("Create", resume);
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Edit(string id)
+        {
+            if (string.IsNullOrWhiteSpace(id))
+            {
+                return NotFound();
+            }
+
+            var resume = await _resumeAppService.GetById(id);
+            ViewData["ResidentAccountId"] = new SelectList(await _resumeAppService.GetAllResidentAccount(), "Id", "FIO", resume.ResidentAccountId);
```

Comment:
Probably can be moved to separate method. Also constants can be used

Reply:
I'll leave it like that for now